### DISCUSSION TITLE
User Stories

### DISCUSSION THREAD

**Discussion by jsapolu99** [Team-Half-and-Half/spirebooks#1] [Ideas]
## 👋 Welcome!
  We’re using this thread as a place to share user stories for our program. We hope that you:
  * Write about features expected/implemented into our program.
  * Ensure you are writing from the perspective of an end user by writing posts using informal natural language.

  To get started, comment below with your very own user story!

**Comment by jsapolu99** [comment]
As a Spire client, I expect to log in to the website and upload my financial information using an intuitive form or file upload system. After submission, I want to view my financial forecasts through dynamic, interactive graphs and charts, which will be updated in response to any audits.

**Comment by beydlern** [comment]
As a Spire accountant, I expect to be able to access the database, view a list of clients, pull a client's data to edit values and visualize the data through dynamic graphs and charts then modify the client's data after a confirmation popup.

**Comment by beydlern** [comment]
As a Spire client, I expect a newly created account to not be able to upload data until the account is verified by an accountant.

**Comment by beydlern** [comment]
As a Spire accountant, I expect to be able to view a list of Spire clients and to be able to toggle their verification status for them be able to upload their data to the website.

**Comment by beydlern** [comment]
As a Spire accountant, I expect a navigation bar so that I am able to easily navigate throughout the website and access important sections swiftly.

**Comment by samallari** [comment]
As a Spire accountant, I expect to be able to see and manage my projections.

**Comment by beydlern** [comment]
As a Spire accountant, I expect to be able to input data into a form that resembles a spreadsheet.

**Comment by beydlern** [comment]
As a Spire client, I expect to be able to view the terms and conditions for this product.

**Comment by beydlern** [comment]
As a Spire accountant, I expect to be able to upload a single xlsm file as a means to upload the data.
